### PR TITLE
[examples] add python to python webcam example using socket.io

### PR DESCRIPTION
**Pull request by djsamseng**
Hey thanks for giving us this library! Just thought this additional example might be useful for those trying to get a barebones implementation together that shows most of the parts from the ground up.

### CODE REVIEW

File: examples/socketio-webcam-videostream/pycli/cli.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    else:
+        webcam = MediaPlayer("/dev/video0", format="v4l2", options=options)
+    if webcam and webcam.audio:
+        pc.addTrack(webcam.audio)
+
+    if webcam and webcam.video:
+        pc.addTrack(webcam.video)
+
+async def createOffer():
+    print("Creating an offer. Waiting for an answer")
```
Comment (rprata) [inline examples/socketio-webcam-videostream/pycli/cli.py:40]: Use `logging` class to print message and add `--verbose` `arg` option in your example.

**Comment by jlaine** [comment]
Hi! I'm not sure I understand what this example does. Is it just changing the signaling method?

**Comment by jlaine** [comment]
@djsamseng Do you think we could have a revised version of this PR which *only* provides a new signaling method, so that it can be plugged into all existing examples? Also, in 2022 what's the advantage of using socket.io as opposed to a plain WebSocket?

**Comment by djsamseng** [comment]
Fair point - I guess this isn't needed then!

**Comment by jlaine** [comment]
I'd still welcome an additional signal method that uses a WebSocket :)

**Pull request closed without merging**